Lib/fontmake/font_project.py

```diff
@@ -20,16 +20,18 @@
 import logging
 import math
 import os
-import plistlib
-import re
 import tempfile
-from io import open
+try:
+    from plistlib import load as readPlist  # PY3
+except ImportError:
+    from plistlib import readPlist  # PY2
+
```

Comment:
Oh, if it's just used in fontmake, I'm ok it's being there. We'd better create a misc.py23 package for fontmake afterwards.